README.md

```diff
@@ -68,23 +70,32 @@ or
 yarn add @guardian/cdk
 ```
 
-Patterns can be imported from the top level of the library (e.g. `import { InstanceRole } from "@guardian/cdk";`) while constructs must be imported from their construct directory (e.g. `import { GuAutoScalingGroup } from "@guardian/cdk/lib/constructs/autoscaling";`)
+Patterns can be imported from the top level of the library:
 
-There are more details on using the CDK library in [docs](docs)
+```typescript
+import { GuScheduledLambda } from "@guardian/cdk";
+```
 
-## Releasing
+While constructs must be imported from their construct directory:
 
-We use [`np`](https://www.npmjs.com/package/np) to help orchestrate the release process.
-To release a new version, run `./script/release`. You will need to be logged in to your `npm` account (`npm login`) which must be part of the Guardian organisation. If you have 2fa enabled, you will be prompted for an OTP during the release process.
+```typescript
+import { GuAutoScalingGroup } from "@guardian/cdk/lib/constructs/autoscaling";
+```
 
-The script automatically prepares a [GitHub Release](https://docs.github.com/en/github/administering-a-repository/about-releases) and opens a pre-populated form in your browser. Please complete the GitHub Release creation manually.
+This is intentional as the patterns ideally solve the majority of use-cases.
+if they don't, PRs are welcome!
```

Comment:
Perhaps we want to encourage users who are working with constructs directly to tell us why they're having to do this too:

```suggestion
If they don't, please let us know about your use-case so that we can consider supporting it via a pattern. Alternatively, PRs are always welcome!
```